src/pages/Portfolio/MyKiva/MyKivaPage.vue

```diff
@@ -207,6 +210,20 @@ apollo.query({ query: myKivaQuery })
 			// eslint-disable-next-line prefer-destructuring
 			activeLoan.value = loans.value[0];
 			fetchLoanUpdates(activeLoan.value.id);
+
+			const preferences = userInfo.value?.userPreferences?.preferences;
+			const formattedPreference = typeof preferences === 'string'
+				? JSON.parse(userInfo.value?.userPreferences?.preferences)
+				: preferences;
+
+			if (!formattedPreference?.myKivaPageExp) {
+				saveUserPreferences({
+					userPreferences: userInfo.value?.userPreferences ?? null,
+					newPreference: {
+						myKivaPageExp: 1,
```

Comment:
We also need to ensure that we update the `ImpactDashboard` component with getting the user preferences, so the user gets navigated to MyKiva like expected. I think that work would be part of this changeset, unless you plan on a quick follow-up.

Reply:
Yes, since right now we only take the user to MyKiva is they have less that 4 loans. We need to have that check OR if they have this user preference saved (they've seen MyKiva before and should continue seeing it).